libethereum/State.cpp

```diff
@@ -502,10 +516,7 @@ size_t State::codeSize(Address const& _a) const
 		return 0;
 }
 
-size_t State::savepoint() const
-{
-	return m_changeLog.size();
-}
+size_t State::savepoint() const { return m_changeLog.size(); }
```

Comment:
In this file some short non-inline function were in single line, some were not. But seeing the changes, I think the opposite option should be better.

Reply:
Allowing one-line only for inline functions inside the class is probably not possible?

Reply:
It is possible. I changed the config to `Inline`. This affects the logging helper function at the beginning of this file.